cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/AbstractParameters.java

```diff
@@ -34,6 +35,7 @@
   static {
     SortedSet<String> validParameterNames = new TreeSet<>(String.CASE_INSENSITIVE_ORDER);
     validParameterNames.add(JSON_PARAM);
+    validParameterNames.add(GET_RESPONSE_SCHEMA);
```

Comment:
we should also update the Java doc for all `Parameters`  class about adding this new parameter(e.g. [RebalanceParameters](https://github.com/linkedin/cruise-control/blob/db6281c2c458ded9dddad72606048bcc01f135ae/cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/parameters/RebalanceParameters.java#L35))

Reply:
Updated all existing java doc under the paramters/ folder. Let me know there are any I missed!